### DISCUSSION TITLE
New to Home Assistant

### DISCUSSION THREAD

**Discussion by speedster-pta** [Dilbert66/esphome-dsckeybus#23] [Q&A]
I'm new to Home Assistant and am working to integrate my DSC 1864. What are the differences between this project and the one by taligentx?

**Comment by Dilbert66** [comment]
This is an ESPHOME front end using the taligentx library.  If you want to use MQTT instead, then stick with the taligentx project which has an MQTT example front

**Reply by speedster-pta** [reply]
Thank you for a great project! I have it installed on my HA now and ironing out the last few bugs. For some reason HA isn't receiving the alarm status from the alarm - it is able to arm/disarm but changes made on the alarm side don't pull through to the HA interface (and automations). Do you know why this might be?

Secondly, is it possible to bypass any zones with this integration?

**Reply by speedster-pta** [reply]
I found the error in my config file - all is working well now. Would still be interested in the bypass functionality though.

**Comment by Dilbert66** [comment]
you can just send *1xx# to bypass a zone where xx is the 2 digit zone number

**Reply by speedster-pta** [reply]
Great stuff. That work's brilliantly. Is there a way to setup sensors to display which zones are bypassed?
